src/components/table/OcTableFiles.vue

```diff
@@ -626,6 +648,10 @@ export default {
       }
     },
     methods: {
+      fileDropped(fileId) {
+        var selectedString = this.selectedIds.join(`, `)
+        alert(selectedString + ` -> ` + fileId);
```

Comment:
Dev leftover? ;)

Reply:
thought of it as a way to show drag & drop result in the docs and how to use it, but I can remove/replace it if you think its not needed or there is a better solution ^^

Reply:
Ah damn it's in the example, sorry 🤦🏽 yeah that's a great idea!